### ISSUE TITLE
JS null read exception navigating video settings with keyboard

### ISSUE THREAD

**Issue by memmam** [SecondFaceSoftware/PortalRevolution-Issues#16]
**Describe the bug**
JS null read exception navigating video settings with keyboard

**To Reproduce**
1. Go to 'Options'
2. Click on 'Video'
3. Highlight the 'Display Mode' dropdown (can use cursor or keyboard)
4. Press 'w' or 'up' on the keyboard
5. See error

**Expected behavior**
Selection should move from 'Display Mode' to the 'Video' tab on the top of the screen

**Screenshots**
![image](https://github.com/SecondFaceSoftware/PortalRevolution-Issues/assets/9062595/b83324c5-98a5-4cf1-9519-df7277aaf182)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - GPU name: Nvidia RTX 4090
 - CPU name: Ryzen 9 7950X
 - RAM (Quantity + Spec): 64GB DDR5-6000

**Comment by memmam** [comment]
It seems like the problem is this code here:

```
    static onMoveToHeaderOrNavbar() {
        const navbarEnabled = $('#ApplyVideoSettingsButton').enabled;
        if(navbarEnabled) {
            $("#ApplyVideoSettingsButton").SetFocus();
        } else {
            $.DispatchEvent("SettingsFocusNavbar");
        }
    }
```

`$('#ApplyVideoSettingsButton').enabled` doesn't appear to exist. If I change the line to `const navbarEnabled = $('#ApplyVideoSettingsButton');` the error goes away, but the conditional never evaluates to true.

I can't find enough digging through the Panorama scripts to have an idea how to fix this given I've never worked with Panorama before, but I thought I'd share what I'd found.

**Comment by StefanH-AT** [comment]
Fixed since 1.0.2
